tests/parsers/benchling_empower/testdata/output/example_01.json

```diff
@@ -0,0 +1,3322 @@
+                                            0.07100000000000001,
+                                            0.07100000000000001,
+                                            0.07200000000000001,
+                                            0.07100000000000001,
+                                            0.06999999999999999,
+                                            0.068,
+                                            0.067,
+                                            0.067,
+                                            0.068,
+                                            0.068,
+                                            0.06899999999999999,
+                                            0.06999999999999999,
+                                            0.07100000000000001,
+                                            0.07200000000000001,
+                                            0.073,
+                                            0.073,
+                                            0.073,
+                                            0.073,
+                                            0.07100000000000001,
+                                            0.06999999999999999,
+                                            0.068,
+                                            0.066,
+                                            0.066,
+                                            0.06499999999999999,
+                                            0.06499999999999999,
+                                            0.066,
+                                            0.067,
+                                            0.06899999999999999,
+                                            0.07100000000000001,
+                                            0.07200000000000001,
+                                            0.073,
+                                            0.07200000000000001,
+                                            0.07100000000000001,
+                                            0.06899999999999999,
+                                            0.068,
+                                            0.068,
+                                            0.068,
+                                            0.068,
+                                            0.06899999999999999
+                                        ]
+                                    ]
+                                }
+                            }
+                        },
+                        {
+                            "device control aggregate document": {
+                                "device control document": [
+                                    {
+                                        "device type": "HPLC"
+                                    }
+                                ]
+                            },
+                            "sample document": {
+                                "sample identifier": "S0103",
```

Comment:
@Preston-O-Benchling, thinking that we should create a new liquid chromatography document for each new sample rather than new measurement document. We'd create a new measurement document then for each measurement of a sample (e.g. absorbance chromatograms, pressure, temperature, etc.). Thoughts?

Reply:
Agreed, @nathan-stender do you have any thoughts on this, could you implement if no blockers? General note for now is that we only have absorbance reported.

Reply:
Makes sense to me! Done!